colossalai/tensor/_ops/addmm.py

```diff
@@ -1,64 +1,60 @@
 import torch
-from typing import Union
 from colossalai.tensor.op_wrapper import colo_op_impl
 from colossalai.nn.layer.parallel_1d._utils import reduce_input, reduce_grad
 from colossalai.tensor import ComputePattern, TensorSpec, ComputePattern, ParallelAction, ColoTensor
 from colossalai.tensor import dist_spec
```

Comment:
I consistently confused the package name with a variable name. Because you tend to use the short names and _ for local variables.
How about name it as distspec?

As [PEP 8](https://www.python.org/dev/peps/pep-0008/#package-and-module-names) clearly describes it:

Package and Module Names

Modules should have short, all-lowercase names. Underscores can be used in the module name if it improves readability. Python packages should also have short, all-lowercase names, although the use of **underscores** is discouraged.

Reply:
So do you mean file names should have no underscore? I think there are many cases like this in colo tensor files.